packages/devtools_app/lib/src/shared/table/table.dart

```diff
@@ -80,6 +80,10 @@ class FlatTable<T> extends StatefulWidget {
   })  : selectionNotifier = selectionNotifier ?? ValueNotifier<T?>(null),
         super(key: key);
 
+  /// List of columns to render.
+  ///
+  /// If different column instances are provided for two instances of the table
+  /// with the same [dataKey], the table will render first instances.
```

Comment:
This isn't true. `dataKey` is used to lookup the `_TableUiState` for the table, which stores sort column index, sort direction, and scroll offset.

Reply:
It is what I observed. The invoked method 'build' belongs to first column of the type, not to column, provided to the table. That is not a problem if the column does not contain state.

Reply:
IIRC what we discussed last week, what you were seeing was a state management bug with when the columns were being defined and updated, but this doesn't have anything to do with the `dataKey` since this `dataKey` is only used to lookup the values I mentioned in my comment above. So we should remove this comment.